bin/git-paste

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Better to add `-o pipefail` to ensure the script exits with non-zero code when `git format-patch` failed.